src/main/java/com/artipie/api/ManageUsers.java

```diff
@@ -109,7 +119,40 @@ public void remove(final String uid) {
             );
             return;
         }
-        throw new IllegalStateException(String.format("User %s does not exist", uid));
+        throw new IllegalStateException(String.format("User %s does not exist", uname));
+    }
+
+    @Override
+    public void alterPassword(final String uname, final JsonObject info) {
+        if (this.users().map(yaml -> yaml.yamlMapping(uname) != null).orElse(false)) {
+            YamlMappingBuilder all = Yaml.createYamlMappingBuilder();
+            final YamlMapping users = this.users().get();
+            for (final YamlNode node : users.keys()) {
+                final String val = node.asScalar().value();
+                if (!uname.equals(val)) {
+                    all = all.add(val, users.yamlMapping(val));
+                }
+            }
+            final YamlMapping user = this.users().get().yamlMapping(uname);
+            YamlMappingBuilder changing = Yaml.createYamlMappingBuilder();
+            for (final YamlNode node : user.keys()) {
+                final String val = node.asScalar().value();
+                if (!ManageUsers.TYPE.equals(val) && !ManageUsers.PASS.equals(val)) {
```

Comment:
Just information, changing is not mandatory.
Line 139: variable name 'val' is not informative, for example it may be 'key' or 'property'